.env

```diff
@@ -5,4 +5,5 @@ NODE_ENV=local
 APP_ENV=Local
 CORS_DOMAIN=*
 ANON_DB_PATH=/Users/braydonharris/dev/personal-projects/tak/server/games_anon.db
-DIST_PATH=dist/
\ No newline at end of file
+DIST_PATH=dist/
+PREVIOUS_FILE=previous.txt
```

Comment:
What do you think about renaming this to `last_used_game_id.txt` or similar?

Reply:
I'm fine either way this was just copied over from the old script.

Reply:
for backwards compatibility going to leave this as is so there is parity between what's used on the server is the same as in the repo.